docs/tutorial/flink.md

```diff
@@ -198,45 +191,105 @@ A [subset of the configuration schema](https://github.com/pangeo-forge/pangeo-fo
 that we just talked about related to inputs/outputs/caches gets dependency injected into the recipe by the runner. 
 
 Various other runner options ([documented here](https://pangeo-forge-runner.readthedocs.io/en/latest/reference/index.html) and [flink-specific here](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/deployment/config/)) 
-can also be passed either in the file configuration format above or directly during CLI `bake` calls. 
+can also be passed either in the file configuration format above or directly during CLI `bake` calls. It's recommended that you use
+the file configuration because some argument values are JSON based.
+
+Here's a quick example of a configuration file and `bake` command slightly more complicated where you're passing [flink-specific configuration options](https://nightlies.apache.org/flink/flink-docs-release-1.16/docs/deployment/config/) and runner options. 
+Note that `-f <path-to-your-runner-config>.<json||py>` would point to the runner configuration file talked about above:
+
+   ```python
+   import json
+   BUCKET_PREFIX = "s3://<bucket-name>/<some-prefix>/"
+   # The storage backend we want
+   s3_fsspec = "s3fs.S3FileSystem"
+   # Credentials for the backend
+   s3_args = {
+       "key": "<your-aws-access-key>",
+       "secret": "<your-aws-access-secret>",
+       "client_kwargs":{"region_name":"<your-aws-bucket-region>"}
+   }
+   # Take note: this is just python. We can reuse these values below
+   c.FlinkOperatorBakery.job_manager_resources = json.dumps({"memory": "2048m", "cpu": 1.0})
```

Comment:
These are all typed as `Dict` (https://github.com/pangeo-forge/pangeo-forge-runner/blob/0401cb03921bcf0b6ba9b779c35a04a0b81768c0/pangeo_forge_runner/bakery/flink.py#L68) so the `json.dumps` will actually cause it to fail. Should be dictionaries!